x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/policy_response.tsx

```diff
@@ -3,8 +3,139 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import React, { memo } from 'react';
+import React, { memo, useMemo } from 'react';
+import styled from 'styled-components';
+import { EuiAccordion, EuiNotificationBadge, EuiHealth } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { htmlIdGenerator } from '@elastic/eui';
+import {
+  HostPolicyResponseActions,
+  HostPolicyResponseActionStatus,
+  HostPolicyResponseConfiguration,
+  Immutable,
+  ImmutableArray,
+} from '../../../../../../common/types';
+import { formatResponse } from './policy_response_friendly_names';
 
-export const PolicyResponse = memo(() => {
-  return <div>Policy Status to be displayed here soon.</div>;
+/**
+ * Nested accordion in the policy response detailing any concerned
+ * actions the endpoint took to apply the policy configuration.
+ */
+const PolicyResponseConfigAccordion = styled(EuiAccordion)`
```

Comment:
Did you test with Kibana's dark mode to see if this looks ok?

Reply:
![image](https://user-images.githubusercontent.com/56409205/81109206-bdbb9c00-8ee7-11ea-8b83-453153c07792.png)

@bfishel  it seems ok to me??

Reply:
noice 🕺